addons/weapons/aceax_models/XtdGearModels.inc

```diff
@@ -34,5 +34,48 @@ class XtdGearModels {
         class Rabbit {};
       };
     };
+    
+    class SWS_Weapon_PDW : SWS_BaseGearModel {
+      label = QNAME(PDW);
+      options[] = {"Body","Variant"};
+      class Body {
+        alwaysSelectable = TRUE;
+        label = "Body Configuration";
+        values[] = {"Sx-90", "SxC-90"};
+        class SX90 {};
+        class SXC90 {};
+      };
+      class Variant {
+        alwaysSelectable = TRUE;
+        label = "Variant";
+        values[] = {"Standard"};
+        class Standard {};
+      };
+    };
+
+    class SWS_Weapon_DMR : SWS_BaseGearModel {
+      label = QNAME(Marksman & Battle Rifles);
+      options[] = {"Body","Underbarrel","Variant"};
+      class Body {
+        alwaysSelectable = TRUE;
+        label = "Body Configuration";
+        values[] = {"SBR-55","SxMR-397"};
+        class SBR55 { description = "SBR-55 Battle Rifle"; };
+        class SxMR397 { description = "Longshot"; };
+      };
+      class Underbarrel {
+        alwaysSelectable = TRUE;
+        label = "Underbarrel Attachment";
+        values[] = {"None", "UBGL"};
+        class None {};
+        class UBGL { description = "UnderBarrel Grenade Launcher."; };
```

Comment:
(useless data)
The funny thing to me is that I probably wrote UBGL because there is a disagreement on if it should be "underslung grenade launcher" or "underBarrel grenade launcher" to the point that not even the actual M203 weapons manual tries to say which it is. I didn't even think about it at the time, but I'll just keep it as UGL for now for clarity's sake.
(/useless data)